src/tools/PrizeTierController/utils/formatPrettyPercentage.ts

```diff
@@ -0,0 +1,3 @@
+export const formatPrettyPercentage = (value: number) => {
+  return (value / 10 ** 7).toLocaleString() + '%'
```

Comment:
Same with the `7` here. 
`return (value / 10 ** DPR_DECIMALS).toLocaleString('en', { style: 'percent' })`
<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString>
<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat>

Reply:
Fixed through 890ab95 and a2e7e14